### PR TITLE
ONe volume test refactor

### PR DESCRIPTION
**Pull request by pauloewerton**
This change adds/edits tests and code refactoring to the ONe volume plugin. To test it: checkout code; configure an ONe cloud on your RAS; run volume requests using a client; run volume unit tests.

### CODE REVIEW

File: src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/volume/v5_4/OpenNebulaVolumePluginTest.java [hunk 1]
```diff
@@ -1,343 +1,393 @@
...
+	// test case: when calling doRequestInstance and receiving a null value (meaning no datastore with enough available
+	// free storage was found) from the getDataStoreId call, then a NoAvailableResources exception should be thrown
+	@Test
+	public void testDoRequestInstanceFail() throws UnexpectedException, InvalidParameterException {
+		// set up
+		CreateVolumeRequest createVolumeRequest = Mockito.spy(this.createVolumeRequest(this.volumeOrder));
+		Mockito.doReturn(null).when(this.plugin).getDataStoreId(Mockito.any(Client.class), Mockito.anyLong());
+
+		// exercise
+        try {
```
Comment (raonismaneoto) [inline src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/volume/v5_4/OpenNebulaVolumePluginTest.java]: the indentation of this test seems weird

File: src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/volume/v5_4/OpenNebulaVolumePluginTest.java [hunk 2]
```diff
@@ -1,343 +1,393 @@
...
+	public void testDoGetInstanceFail() throws UnexpectedException {
+		// set up
+		ImagePool imagePool = Mockito.mock(ImagePool.class);
+
+		Mockito.when(OpenNebulaClientUtil.getImagePool(Mockito.any(Client.class))).thenReturn(imagePool);
+		Mockito.when(imagePool.getById(Mockito.anyInt())).thenReturn(null);
 
 		// exercise
-		this.plugin.deleteInstance(volumeOrder, cloudUser);
+        try {
```
Comment (raonismaneoto) [inline src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/volume/v5_4/OpenNebulaVolumePluginTest.java]: same thing

File: src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/volume/v5_4/OpenNebulaVolumePluginTest.java (hunk 1 above)
Comment (pauloewerton) [reply src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/volume/v5_4/OpenNebulaVolumePluginTest.java]: i attempted to fix it, let's see if it works. ;-)

**Comment by pauloewerton** [comment]
@raonismaneoto thank you for the review. comments addressed.

**Pull request merged**